src/BookNavigator/search/a-search-result.js

```diff
@@ -1,6 +1,6 @@
-import { nothing } from 'lit-html';
-import { html, LitElement } from 'lit-element';
-import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import { nothing } from 'lit';
+import { html, LitElement } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html';
```

Comment:
Apparently `lit/directives` needs `.js` at the end :/ This is causing some issues in the build step: https://app.netlify.com/sites/lucid-poitras-9a1249/deploys/620cbb1b0da87400072c24bf#L74

Add .js here and anywhere `lit/directives` is imported from.

```suggestion
import { unsafeHTML } from 'lit/directives/unsafe-html.js';
```